west.yml

```diff
@@ -34,100 +34,156 @@ manifest:
       path: modules/hal/altera
+      groups:
+        - hal
     - name: hal_atmel
       revision: d17b7dd92d209b20bc1e15431d068edc29bf438d
       path: modules/hal/atmel
+      groups:
+        - hal
     - name: hal_cypress
       revision: 81a059f21435bc7e315bccd720da5a9b615bbb50
       path: modules/hal/cypress
+      groups:
+        - hal
     - name: hal_espressif
       revision: a5d7cb9e6676b08b15ac5b0c54a98514d8bc257c
       path: modules/hal/espressif
+      groups:
+        - hal
       west-commands: west/west-commands.yml
       revision: c21d252a2188fcb60d52dceb6b018a9f4a371817
       path: modules/hal/espressif
+      groups:
+        - hal
     - name: hal_infineon
       revision: f1fa8241f8786198ba41155413243de36ed878a5
       path: modules/hal/infineon
+      groups:
+        - hal
     - name: hal_microchip
       revision: b280eec5d3b1296b231117c1999bcd0269b6ecc4
       path: modules/hal/microchip
+      groups:
+        - hal
     - name: hal_nordic
       revision: 74b3b21f60aa3dc9a4364ffc28dbb47ad8b699a9
       path: modules/hal/nordic
+      groups:
+        - hal
     - name: hal_nuvoton
       revision: b4d31f33238713a568e23618845702fadd67386f
       path: modules/hal/nuvoton
+      groups:
+        - hal
     - name: hal_nxp
       revision: ac1ad8d44a407f8f1fe813e983bda085e9ef342a
       path: modules/hal/nxp
+      groups:
+        - hal
     - name: hal_openisa
       revision: 40d049f69c50b58ea20473bee14cf93f518bf262
       path: modules/hal/openisa
+      groups:
+        - hal
     - name: hal_quicklogic
       revision: b3a66fe6d04d87fd1533a5c8de51d0599fcd08d0
       path: modules/hal/quicklogic
       repo-path: hal_quicklogic
+      groups:
+        - hal
     - name: hal_silabs
       revision: be39d4eebeddac6e18e9c0c3ba1b31ad1e82eaed
       path: modules/hal/silabs
+      groups:
+        - hal
     - name: hal_st
       revision: 575de9d461aa6f430cf62c58a053675377e700f3
       path: modules/hal/st
+      groups:
+        - hal
     - name: hal_stm32
       revision: 91054cd84aaffa260efcbf3b591badcc4fa013ab
       path: modules/hal/stm32
+      groups:
+        - hal
     - name: hal_ti
       revision: 3da6fae25fc44ec830fac4a92787b585ff55435e
       path: modules/hal/ti
+      groups:
+        - hal
     - name: hal_xtensa
       revision: 2f04b615cd5ad3a1b8abef33f9bdd10cc1990ed6
       path: modules/hal/xtensa
+      groups:
+        - hal
     - name: libmetal
       revision: 39d049d4ae68e6f6d595fce7de1dcfc1024fb4eb
       path: modules/hal/libmetal
+      groups:
+        - hal
     - name: littlefs
       path: modules/fs/littlefs
+      groups:
+        - fs
       revision: 9e4498d1c73009acd84bb36036ee5e2869112a6c
     - name: loramac-node
       revision: 12019623bbad9eb54fe51066847a7cbd4b4eac57
       path: modules/lib/loramac-node
     - name: lvgl
       revision: 31acbaa36e9e74ab88ac81e3d21e7f1d00a71136
       path: modules/lib/gui/lvgl
+      groups:
+        - gui
```

Comment:
Is it really worth having a group with just one project in it?

Same question for the `bootloader`, `audio`, `bsim_hw_models` groups.

Reply:
done